docs/src/integrate/patina_dxe_core_requirements.md

```diff
@@ -165,6 +165,40 @@ Resource Descriptor HOB v2 describing it.
 > Platforms must create Resource Descriptor HOB v2s for all memory resources including MMIO and reserved memory with
 > a valid cacheability attribute set.
 
+#### 2.2.1 Resource Descriptor Hob v2 Guidance
+
+Resource Descriptor HOBs need to define the system's memory map. Historically, some platforms have only described memory that is in use, but the HOBs should describe the system memory map, as it stands, at
+the end of PEI. This includes caching attributes, so that Patina is able to consume the data and provide
+enhanced memory protections from the start of DXE.
+
+##### PEI Video Region Example
+
+By the start of DXE, most system memory should be configured as Write Back (EFI_MEMORY_WB), and most Memory Mapped I/O
+should be configured as uncached (EFI_MEMORY_UC). There are exceptions to this, based upon platform level decisions.
+For example, if the pre-DXE phase configured a video device and displayed something on the screen, then the memory
+region associated with the frame buffer, for performance, should be reflected for that region of memory.
+
+##### Firmware Devices
+
+SPI flash accessiable through MMIO is another complex example.
+
+Consider the following scenario:
```

Comment:
I think this should cover cache type for HPET and SPI flash. HPET -> UC. SPI flash -> WP (including var store for early read perf).